src/selectors/entities/emojis.js

```diff
@@ -44,3 +46,11 @@ export const getCustomEmojiIdsSortedByName: (state: GlobalState) => Array<string
         );
     }
 );
+
+export const existsInCustomEmojis: (state: GlobalState, emojiName: string) => boolean = createSelector(
+    getCustomEmojisByName,
+    (_, emojiName) => emojiName,
+    (customEmojisByName, emojiName) => {
+        return doesMatchNamedEmoji(`:${emojiName}:`) && customEmojisByName.has(emojiName);
```

Comment:
Curious, what's the point of the regex here? Wouldn't checking `customEmojisByName.has(emojiName)` do everything you need and have better performance than matching a regex?

Also making this a selector isn't going to help performance-wise as the "emojiName" argument is going to change often. If you care about the memoization you would need to follow the `makeExistsInCustomEmojis` pattern where it returns a function that creates a selector

Reply:
Thanks Joram, I have looked into this again and this PR is not necessary.  I'm closing this now, and will explain more on the webapp PR.